merge_coda_files/merge_coda_files.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import time
+from os import path
+import json
+from dateutil.parser import isoparse
+
+from core_data_modules.cleaners import Codes
+from core_data_modules.cleaners.cleaning_utils import CleaningUtils
+from core_data_modules.traced_data import Metadata
+from core_data_modules.traced_data.io import TracedDataJsonIO, TracedDataCoda2IO
+from core_data_modules.util import IOUtils
+from core_data_modules.data_models import Scheme
+
+# opens a coding scheme
+def _open_scheme(filepath):
+    with open(filepath, "r") as f:
+        firebase_map = json.load(f)
+        return Scheme.from_firebase_map(firebase_map)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Merges manually cleaned files back into a traced data'
+        'file.')
+    parser.add_argument('user', help='User launching this program')
+    parser.add_argument('json_input_path', metavar='json-input-path',
+                        help='Path to JSON input file TracedData objects')
+    parser.add_argument('variable_name', metavar='variable-name',
+                        help='Name of message variable in flow')
+    parser.add_argument('coda_input_path', metavar='coda-input-path',
+                        help='Manually-coded Coda file')
+    parser.add_argument('coding_scheme_path', metavar='coding-scheme-path',
+                        help='Path to Coda scheme file used on coda file')
+    parser.add_argument('traced_json_output_path', metavar='json-output-path',
+                        help='Path to a JSON file to write merged results to')
+    parser.add_argument('is_yes_no', metavar='is-yes-no',
+                        help='Is this variable a yes no question')
```

Comment:
Can you add a comment/help string* explaining why this needs to be treated differently

*whichever is more appropriate

Reply:
I don't quite understand what you mean here.